app/serializers.py

```diff
@@ -84,3 +84,31 @@ def update(self, instance, validated_data):
         instance.availability = validated_data.get('availability', instance.availability)
         instance.save()
         return instance
+
+
+class RequestSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Request
+        fields = [
+            'skill',
+            'description',
+            'mentee',
+            'mentor',
+            'status'
+        ]
+
+    def update(self, instance, validated_data):
+        skill = validated_data.get('skill')
+        if skill:
+            instance.skill.add(*skill)
+
+        description = validated_data.get('description')
+        if description:
+            instance.description.add(*description)
+
+        instance.mentor = validated_data.get('mentor', instance.mentor)
```

Comment:
I don't think we want to make it possible for people to update the mentee(creator) of a request. We will like to update the mentor when the request has been fulfilled.

Reply:
@Del-sama Not sure i know how to go about this...I might need some help here please.